src/components/Common/index.tsx

```diff
@@ -45,3 +45,30 @@ export const OpacityHoverState = css`
     },
   }) => `opacity ${duration.medium} ${timing.ease}`};
 `
+
+// eslint-disable-next-line no-use-before-define
```

Comment:
even though this is clearly imported eslint fails my build

Reply:
Can you ping @vm and see why this error is occurring?